hs_collection_resource/utils.py

```diff
@@ -30,3 +39,112 @@ def add_or_remove_relation_metadata(add=True, target_res_obj=None, relation_type
 
     if set_res_modified:
         resource_modified(target_res_obj, last_change_user)
+
+
+def update_collection_list_csv(collection_obj, keep_local_csv=False):
+    """
+    This func is to create a new csv file in bag that lists info of all contained resources
+    :param collection_obj: collection resource object
+    :param keep_local_csv: (default: False) if True, return the full path to csv on django server,
```

Comment:
@pkdash well, in real use this function will upload the local csv file to irods then remove it by default. So this "keep_local_csv" is a switch to control this "remove" action as in test case we dont want to remove it.